python_modules/libraries/dagster-azure/dagster_azure/adls2/io_manager.py

```diff
@@ -62,6 +62,10 @@ def unlink(self, path: UPath) -> None:
         with self._acquire_lease(file_client, is_rm=True) as lease:
             file_client.delete_file(lease=lease, recursive=True)
 
+    def make_directory(self, path: UPath) -> None:
+        # It is not necessary to create directories in S3
```

Comment:
can you update this comment to say `ADLS2` instead of `S3`?

Reply:
Sorry, my bad. Should be good now.